src/steps/writing/creation/dotVSCode.ts

```diff
@@ -50,5 +60,17 @@ export async function createDotVSCode(options: Options) {
 			"eslint.rules.customizations": [{ rule: "*", severity: "warn" }],
 			"typescript.tsdk": "node_modules/typescript/lib",
 		}),
+		"tasks.json": await formatJson({
+			tasks: [
+				{
+					detail: "Build the project",
+					label: "build",
+					problemMatcher: [],
+					script: "build",
+					type: "npm",
+				},
+			],
+			version: "2.0.0",
+		}),
```

Comment:
[Bug] You'll need to also commit a `.vscode/tasks.json` to this repo. The migration test makes sure changes to templates also get applied to the repo.

Reply:
Okay cool - resolved. Quirky!

Reply:
That's a nice way of putting it 😄